cmd/argocd/commands/root.go

```diff
@@ -94,3 +97,40 @@ func NewCommand() *cobra.Command {
 
 	return command
 }
+
+// HandleCommandExecutionError processes the error returned from executing the command.
+// It handles both standard Argo CD commands and plugin commands. We don't require to return
+// error but we are doing it to cover various test scenarios.
+func HandleCommandExecutionError(err error, isArgocdCLI bool, o ArgoCDCLIOptions) error {
+	cli.SetLogFormat("text")
+	cli.SetLogLevel("info")
```

Comment:
```suggestion
	initConfig()
```

I would also add a comment that this is required because, at this point, the execution of the command has failed, which is what `cobra.OnInitialize()` needs to trigger.